packages/server-wallet/src/models/ledger-proposal.ts

```diff
@@ -0,0 +1,74 @@
+import {JSONSchema, Model, TransactionOrKnex} from 'objection';
+import {Address, SimpleAllocation} from '@statechannels/wallet-core';
+
+import {Bytes32} from '../type-aliases';
+
+interface RequiredColumns {
+  readonly channelId: Bytes32;
+  readonly proposal: SimpleAllocation | null;
```

Comment:
Why should this column ever take the value `null`?

Reply:
When we want to explicitly remove the proposal, which is an indication of the protocol having "dealt with" it

Reply:
Why not have a `dealth_with: boolean` column? (by a more meaningful name)

Reply:
We can certainly do this, I guess I had preferred deleting data we no longer needed

Reply:
We can still delete rows where `dismissed: true`, if we wish. It would be sensible to do so when the next state becomes supported, for instance.